.github/workflows/push-image.yaml

```diff
@@ -61,6 +62,13 @@ jobs:
       - name: Store the first line of commit message
         id: commit_message
         run: echo "value=$(echo "${{ github.event.head_commit.message }}" | head -n 1)" >> $GITHUB_OUTPUT
+      - name:  Store the migration_required value to output
+        uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36 # v3.0.2
+        id: changes
+        with:
+          filters: |
+            migration:
+              - 'migration/**'
```

Comment:
I'm using the `dorny/paths-filter` action, which is great for checking for changes in directories or files.